pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -71,6 +71,36 @@ func ValidateKubeSchedulerConfiguration(cc *config.KubeSchedulerConfiguration) f
 		allErrs = append(allErrs, field.Invalid(field.NewPath("podMaxBackoffSeconds"),
 			cc.PodMaxBackoffSeconds, "must be greater than or equal to PodInitialBackoffSeconds"))
 	}
+
+	binders := 0
```

Comment:
prefer to refactor out the existing checks from the other function to use them here